src/components/canrawview/canrawview_p.h

```diff
@@ -90,29 +102,33 @@ private slots:
         emit q->dockUndock();
     }
 
-    void update(QList<int> &clickedIndexes)
-    {
-    }
+    void update() {}
 
-    void sort(const int currentIndex, QList<int> &clickedIndexes)
+    void sort(const int clickedIndex)
     {
-        clickedIndexes.removeFirst();
-        clickedIndexes.append(currentIndex);
+        int currentSortOrder = ui->tv->horizontalHeader()->sortIndicatorOrder();
+        int sortIndex = clickedIndex;
+        static int prevIndex = 0;
```

Comment:
Each instance have dedicated private class so it shouldn't be problem. However I would prefer to have it as class member (private) instead. Declaring prevIndex as static will put it to global data section (not sure but BSS probably). Declaring it as class member will keep it together with class. Processors likes to access "linear" memory :)